python/utilities/MixingTimeResolution_Aux.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include <pybind11/pybind11.h>
+
+#include <goofit/PDFs/physics/MixingTimeResolution_Aux.h>
+
+using namespace GooFit;
+namespace py = pybind11;
+
```

Comment:
The tricky part about a pure virtual function is that the class cannot be instanciated if it contains a pure virtual function (that's the point of a pure v.f.). So you have to make a special C++ class that just adds non-pure functions (using a PyBind macro) so that you can bind it (since PyBind instanciates the class as part of preparing the python version).